frontend/packages/dev-console/src/components/add/AddPageLayout.tsx

```diff
@@ -54,10 +54,13 @@ const AddPageLayout: React.FC<AddPageLayoutProps> = ({ title, hintBlock: additio
                 onChange={(checked) => {
                   setShowDetails(checked);
                 }}
-                data-test-id="odc-add-page-details-switch"
+                data-test="details-switch"
               />
             </Tooltip>
-            <span className="odc-add-page-layout__hint-block__details-switch__text">
+            <span
+              className="odc-add-page-layout__hint-block__details-switch__text"
+              data-test="details-label"
+            >
```

Comment:
`details-switch` and `details-label` doesn't match our meeting result.

Should we add a div here which has data-test="details" and then "switch" and "label" to the child elements?

Reply:
Changed it to

`details-options` > `switch`
`details-options` > `label`